python/lib/sift_py/ingestion/channel.py

```diff
@@ -225,70 +256,73 @@ def from_pb(cls, val: channel_pb.ChannelDataType.ValueType) -> "ChannelDataType"
 
-        if val == "CHANNEL_DATA_TYPE_DOUBLE" or val == ChannelDataTypeStrRep.DOUBLE.value:
+        if val == ChannelDataTypeStrRep.DOUBLE:
             return cls.DOUBLE
-        elif val == "CHANNEL_DATA_TYPE_STRING" or val == ChannelDataTypeStrRep.STRING.value:
+        elif val == ChannelDataTypeStrRep.STRING:
             return cls.STRING
-        elif val == "CHANNEL_DATA_TYPE_ENUM" or val == ChannelDataTypeStrRep.ENUM.value:
+        elif val == ChannelDataTypeStrRep.ENUM:
             return cls.ENUM
-        elif val == "CHANNEL_DATA_TYPE_BIT_FIELD" or val == ChannelDataTypeStrRep.BIT_FIELD.value:
+        elif val == ChannelDataTypeStrRep.BIT_FIELD:
             return cls.BIT_FIELD
-        elif val == "CHANNEL_DATA_TYPE_BOOL" or val == ChannelDataTypeStrRep.BOOL.value:
+        elif val == ChannelDataTypeStrRep.BOOL:
             return cls.BOOL
-        elif val == "CHANNEL_DATA_TYPE_FLOAT" or val == ChannelDataTypeStrRep.FLOAT.value:
+        elif val == ChannelDataTypeStrRep.FLOAT:
             return cls.FLOAT
-        elif val == "CHANNEL_DATA_TYPE_INT_32" or val == ChannelDataTypeStrRep.INT_32.value:
+        elif val == ChannelDataTypeStrRep.INT_32:
             return cls.INT_32
-        elif val == "CHANNEL_DATA_TYPE_INT_64" or val == ChannelDataTypeStrRep.INT_64.value:
+        elif val == ChannelDataTypeStrRep.INT_64:
             return cls.INT_64
-        elif val == "CHANNEL_DATA_TYPE_UINT_32" or val == ChannelDataTypeStrRep.UINT_32.value:
+        elif val == ChannelDataTypeStrRep.UINT_32:
             return cls.UINT_32
-        elif val == "CHANNEL_DATA_TYPE_UINT_64" or val == ChannelDataTypeStrRep.UINT_64.value:
+        elif val == ChannelDataTypeStrRep.UINT_64:
             return cls.UINT_64
-
-        return None
-
-    def as_human_str(self) -> str:
-        if self == self.__class__.DOUBLE.value:
-            return ChannelDataTypeStrRep.DOUBLE.value
-        elif self == self.__class__.STRING.value:
-            return ChannelDataTypeStrRep.STRING.value
-        elif self == self.__class__.ENUM.value:
-            return ChannelDataTypeStrRep.ENUM.value
-        elif self == self.__class__.BIT_FIELD.value:
-            return ChannelDataTypeStrRep.BIT_FIELD.value
-        elif self == self.__class__.BOOL.value:
-            return ChannelDataTypeStrRep.BOOL.value
-        elif self == self.__class__.FLOAT.value:
-            return ChannelDataTypeStrRep.FLOAT.value
-        elif self == self.__class__.INT_32.value:
-            return ChannelDataTypeStrRep.INT_32.value
-        elif self == self.__class__.INT_64.value:
-            return ChannelDataTypeStrRep.INT_64.value
-        elif self == self.__class__.UINT_32.value:
-            return ChannelDataTypeStrRep.UINT_32.value
-        elif self == self.__class__.UINT_64.value:
-            return ChannelDataTypeStrRep.UINT_64.value
+        else:
+            raise Exception("Unreachable")
+
+    def as_human_str(self, api_format: bool = False) -> str:
```

Comment:
This recommendation (https://github.com/sift-stack/sift/pull/102#discussion_r1801562128) worked for the other types but not `ChannelDataType` since it uses `ChannelDataTypeStrRep`.